Lib9c/Delegation/Delegation.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Numerics;
+using Libplanet.Crypto;
+using Libplanet.Types.Assets;
+
+namespace Nekoyume.Delegation
+{
+    public class Delegation
```

Comment:
If there are no properties or methods to override, I think it's a good idea to seal the class. 😀